lib/Doctrine/DBAL/Migrations/Version.php

```diff
@@ -372,26 +382,21 @@ public function __toString()
 
     private function executeRegisteredSql($dryRun = false, $timeAllQueries = false)
     {
+        $executedQueries = 0;
```

Comment:
before or after the loop does not change anything. The loop does not mutate the array